### ISSUE TITLE
www.data18.com/

### ISSUE THREAD

**Issue by eduardomtz28** [adultmm/AdultMediaManager#18]
https://www.data18.com/scenes
here are all the sites and movies I would not have to add more consider it

**Comment by chaitrakiran2010** [comment]
Please add database of https://www.data18.com, good for old collection. new data18 scrapers not working

**Comment by adultmm** [comment]
Unfortunately, there is a problem with the data18.com site. It doesn't serve the images without the appropriate referrer, so the images on the site cannot be hot-linked, and used in EMM. You can try it yourself, just copy the URL of any poster to another browser window. So we should have a server to proxy the images on this site. Something like this: https://myimageproxy.com?image=https://www.data.18/image-from-the-site-jpg&use-referer=https://www.data.18 The image proxy download the image with the given referrer and return it. It's not a big deal to write a proxy like this, but I have no server to run the code. 

Does anybody know an online service for this task?

I'm thinking about the solution...

**Comment by chaitrakiran2010** [comment]
Yes I agree with you. at least implement information (nfo) and others (Actors name, studio) except images. when we click on  Continue reading on original site...it takes us to original site from there you can get images (here you can do you magic).

**Comment by chaitrakiran2010** [comment]
Please consider data18.com database as a separate project. You just go on with your Adult Media Manager, The Best Adult Media Scraper in my opinion. Add more adult sites. I am going to send you a huge list.

**Comment by chaitrakiran2010** [comment]
Forget data18.com database, as it is incomplete. They don't have complete data for babes.com, NaughtyAmerica.com so many sites....... Please add scrapper for these sites. 
FantasyMassage 
  - AllGirlMassage
  - FantasyMassage
  - MassageParlor
  - MilkingTable
  - NuruMassage
  - POVMassage
  - SoapyMassage
  - TrickySpa

**Comment by adultmm** [comment]
Data18.com meta-scaper is now available in v0.10.0-BETA.1, also FantasyMassage sites.

**Comment by chaitrakiran2010** [comment]
Thank you my friend. I don't have any words for your excellent job and v0.10.0-BETA.1 is working perfectly.
